### PR TITLE
Added new mods

### PR DESCRIPTION
**Pull request by Rafii2198**
Depends on #51 
Again, everything is tested for compatibility. No crashes, no issues.

### CODE REVIEW

File: versions.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-fabricVersion: 0.14.22
+fabricVersion: 0.14.23
```
Comment (kristofbolyai) [inline versions.yml]: Roll this back.

File: versions.yml [hunk 2]
```diff
@@ -331,6 +346,13 @@ modGroups:
       desc: Allows resource packs provided by servers to be overridden and moved.
       url: https://cdn.modrinth.com/data/PiuygVWJ/versions/lRVqr2EB/spu-1.7.jar
 
+    - name: Midnight Controlls
```
Comment (kristofbolyai) [inline versions.yml:357]: I dont think this is a hud enchancement

Comment (kristofbolyai) [reply versions.yml:357]: Nvm, I see this is a misc.

**Review by kristofbolyai — COMMENTED** [review]
Add back wynnlib and we are good to merge.

**Comment by Rafii2198** [comment]
Added

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**